Week3/.env

```diff
@@ -0,0 +1 @@
+FIREBASE_SECRET_KEY="./Firebase/first-assignment-99ab9-firebase-adminsdk-vigov-ea6b693df0.json"
```

Comment:
Do not push .env file to remote repository, instead add this file in .gitignore

Reply:
yes , i did that but git somehow gitignore was not taking it in that,so i deleted the file because it was not of the use